manifests/r/Rainmeter/Rainmeter/4.5.17/Rainmeter.Rainmeter.installer.yaml

```
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS2.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Rainmeter.Rainmeter
+PackageVersion: 4.5.17
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
```

```suggestion
- Architecture: x64
```
Task Manager shows the architecture for the app is x64.